js/app/src/components/Audio/Audio.test.ts

```diff
@@ -18,10 +20,14 @@ const loading_status = {
 };
 
 describe("Audio", () => {
+	beforeAll(() => {
+		window.HTMLMediaElement.prototype.play = vi.fn();
+		window.HTMLMediaElement.prototype.pause = vi.fn();
+	});
 	afterEach(() => cleanup());
 
 	test("renders provided value and label", async () => {
-		const { getByTestId, queryAllByText } = render(Audio, {
+		const { getByTestId, queryAllByText } = await render(Audio, {
```

Comment:
`render` is now async because it ensure the DOM is updated before resolving. We need to await it to ensure this has all happened.

Reply:
Interesting - the same function can be be simultaneously async and not async? 🤯